moped-database/migrations/1693933008471_more_atsd_project_tags/up.sql

```diff
@@ -0,0 +1,36 @@
+INSERT INTO moped_tags (name, type, slug, is_deleted) VALUES
+    ('FHWA Experimentation', 'Other', 'fhwa_experimentation', TRUE),
```

Comment:
are you inserting deleted tags because the access migration needs the tags, but no users should use them going foward?

Reply:
yes, that's right! these tags are no longer active.